packages/terra-functional-testing/src/docker/docker-compose.yml

```diff
@@ -1,10 +1,16 @@
-version: '3.7'
```

Comment:
This file has been converted from the [docker-swarm](https://github.com/SeleniumHQ/docker-selenium/tree/selenium-3#version-3-with-swarm-support) example to the [docker-compose example](https://github.com/SeleniumHQ/docker-selenium/tree/selenium-3#version-3) as recommended by docker-selenium. A [healthcheck](https://github.com/SeleniumHQ/docker-selenium/tree/selenium-3#adding-a-healthcheck-to-the-grid) was also added